src/tests/Microsoft.PowerFx.Core.Tests.Shared/ExpressionTestCases/Join.txt

```diff
@@ -0,0 +1,81 @@
+﻿#SETUP: PowerFxV1CompatibilityRules,StronglyTypedBuiltinEnums
```

Comment:
Do we need the 'StronglyTypedBuiltinEnums' setup for this test? If not, we should remove it.

Reply:
We need it.

Reply:
For all tests, or only for a few of them? If the latter, we should split it out into different test files.